### PR TITLE
Provide photo credit to unsplash component

### PR DESCRIPTION
**Pull request by moltenkaizen**
Creating this PR to respond to PR #5

### CODE REVIEW

File: src/components/Unsplash-Background.vue [hunk 1]
```diff
@@ -20,16 +32,20 @@
...
+        headers: new Headers({
+          Authorization: 'Client-ID b325d6aa7cdaacc3b279aa500e0f091c1e3ae2b986270547a005dfe1835b78a9',
+        })
       });
       fetch(request)
         .then(response => response.json())
         .then((response) => {
-          this.backgroundImage = response.urls.regular;
-          document.body.style.backgroundImage = `url(${this.backgroundImage})`;
+          this.backgroundImage.location = '';
```
Comment (jacobarriola) [inline src/components/Unsplash-Background.vue]: Looks like you're adding this below. Not sure if this was an oversight 🤔

File: src/components/Unsplash-Background.vue [hunk 2]
```diff
@@ -8,9 +15,14 @@
     data() {
       return {
         colors: ['#1abc9c', '#2ecc71', '#3498db', '#9b59b6', '#34495e', '#16a085', '#27ae60',
-        '#2980b9', '#8e44ad', '#2c3e50', '#f1c40f', '#e67e22', '#e74c3c', '#95a5a6', '#f39c12',
-        '#d35400', '#c0392b', '#7f8c8d'],
-        backgroundImage: ''
+          '#2980b9', '#8e44ad', '#2c3e50', '#f1c40f', '#e67e22', '#e74c3c', '#95a5a6', '#f39c12',
+          '#d35400', '#c0392b', '#7f8c8d'],
+        backgroundImage: {
```
Comment (jacobarriola) [inline src/components/Unsplash-Background.vue:20]: Nice!!

**Review by jacobarriola — COMMENTED** [review]
Looking good! The only thing I see is a possible dup of the `backgroundImage.location`.

File: src/components/Unsplash-Background.vue (hunk 2 above)
Comment (moltenkaizen) [reply src/components/Unsplash-Background.vue:20]: @jacobarriola Cleaned up that due, works fine when I test.

**Pull request merged**